common/app/views/support/Profile.scala

```diff
@@ -21,7 +21,7 @@ sealed trait ElementProfile {
   def autoFormat: Boolean
 
   def elementFor(image: ImageMedia): Option[ImageAsset] = {
-    val sortedCrops = image.imageCrops.sortBy(-_.width)
```

Comment:
I think intentional, possibly due to some usage I haven't considered yet. Will make sure to find out if we decide to go down this route.